amulet/api/chunk/chunk.py

```diff
@@ -387,3 +389,14 @@ def misc(self) -> dict:
     def misc(self, misc: dict):
         assert isinstance(misc, dict), "misc must be a dictionary."
         self._misc = misc
+
+    @property
+    def set_light_off(self) -> bool:
```

Comment:
This feature only exists in Java. I would prefer if this was left in the misc data.
The rewrite gives each format the ability to define its own chunk class so in the future we could have something like this but for now it should be left in misc.